pkg/pushsync/pushsync.go

```diff
@@ -126,19 +130,34 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 		return swarm.ErrInvalidChunk
 	}
 
-	span, _, ctx := ps.tracer.StartSpanFromContext(ctx, "pushsync-handler", ps.logger, opentracing.Tag{Key: "address", Value: chunk.Address().String()})
-	defer span.Finish()
-
-	// Get price we charge for upstream peer read at headler
+	// Get price we charge for upstream peer read at header.
 	responseHeaders := stream.ResponseHeaders()
 	price, err := headerutils.ParsePriceHeader(responseHeaders)
-
 	if err != nil {
-		// if not found in returned header, compute the price we charge for this chunk and
```

Comment:
maybe the 'and' can be removed from the sentence as it is unnecessary, but please keep the comment